lib/modules/apostrophe-docs/lib/api.js

```diff
@@ -3,6 +3,12 @@ var _ = require('@sailshq/lodash');
 var Promise = require('bluebird');
 
 module.exports = function(self, options) {
+  self.trashPrefixFields = [ 'slug' ];
```

Comment:
This should move to apostrophe-doc-type-manager so that each doc type can have its own array.